app/build.gradle

```diff
@@ -105,6 +105,7 @@ android {
     aaptOptions {
         cruncherEnabled = false
     }
+    buildToolsVersion = '29.0.2'
```

Comment:
Why?

Reply:
That got changed automatically  
when i was setting up the project on my system .
Should i remove it ?

Reply:
yes